app/controllers/publications_controller.rb

```diff
@@ -1,54 +0,0 @@
-class PublicationsController < DocumentsController
-  enable_request_formats index: [:json, :atom]
-  before_filter :expire_cache_when_next_publication_published
-  before_filter :redirect_statistics_filtering, only: [:index]
-  before_filter :redirect_statistics_documents, only: [:show]
-
-  def index
```

Comment:
We can't remove this as I think we'll still need to provide the `.atom` feed of publications.